.github/workflows/ci.yml

```diff
@@ -158,21 +182,34 @@ jobs:
           python-version: 3.6
 
       - name: Build wheels
+        if: startsWith(matrix.os, 'ubuntu')
         run: |
           python -m pip install cibuildwheel==1.7.4
           python -m cibuildwheel python --output-dir python/wheelhouse
         env:
-          CIBW_BEFORE_ALL: python/tools/prepare_build_environment.sh
+          CIBW_BEFORE_ALL: python/tools/prepare_build_environment_linux.sh
           CIBW_BEFORE_BUILD: pip install -r python/install_requirements.txt
           CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
           CIBW_BEFORE_TEST: python/tools/prepare_test_environment.sh
           CIBW_TEST_COMMAND: '{project}/python/tools/run_test.sh {project}'
           CIBW_SKIP: cp27-* pp* *i686
 
+      - name: Build wheels
+        if: startsWith(matrix.os, 'macos')
+        run: |
+          python -m pip install cibuildwheel==1.7.4
+          python -m cibuildwheel python --output-dir python/wheelhouse
+        env:
+          CIBW_BEFORE_ALL: python/tools/prepare_build_environment_macos.sh
+          CIBW_BEFORE_BUILD: pip install -r python/install_requirements.txt
+          CIBW_BEFORE_TEST: python/tools/prepare_test_environment.sh
+          CIBW_TEST_COMMAND: '{project}/python/tools/run_test.sh {project}'
+          CIBW_SKIP: cp27-* pp* *i686
```

Comment:
Is there any way to factorize this block with the Linux step? Looks like only `CIBW_BEFORE_ALL` is different.

Reply:
I've used platform-specific variants instead.